engine/share/includes/scr14_c.inc

```diff
@@ -22,9 +22,9 @@ Copyright>        software under a commercial license.  Contact Altair to discus
 Copyright>        commercial version may interest you: https://www.altair.com/radioss/.    
 C.../SCR14R/
 C...mis a 0 par defaut en spmd
-      COMMON /SCR14R/ DTANIM0,TANIM0,DTANISENS,TANIMSENS
+      COMMON /SCR14R/ DTANIM0,TANIM0,DTANISENS,TANIMSENS,TANIM_STOP0
```

Comment:
Hi @Schwoertzig, 

Could you use a module for your option, instead of adding variables in common blocks?

On the long run, we want to get rid of common blocks.
Thank you

Reply:
Hi @laurent-altr ,
This enhancement is a new parameter related to the same option /ANIM/DT. All other parameters are also in the same COMMON statement. So it is not relevant to add a module just for this new parameter. All parameters must be in the same module.  May I suggest to add this new parameter now and to request an indepedent work to introduce the module for ANIM option (dedicated Pull Request).

Reply:
Hi, 
- as requested, I introduced a new module ; I called it anim_mod.F. My new variables TANIM_STOP and TANIM_STOP0 are now in this new module.
- I also moved all Real variables related to animation files from include to this module (TANIM, DTANIM...).  Because it was disturbing to have a part of them in scr14_c.inc/com06_c.inc and the other part in anim_mod.F. 
-  If the aim is to get rid of common blocks, we can do the same with integer related to animation file. But in a second step because changes will be more readable and several source files are affected.